### ISSUE TITLE
V1.0.781 preventing other items on the form from refreshing

### ISSUE THREAD

**Issue by ukandrewc** [MicrosoftEdge/WebView2Feedback#875]
**Description**
WebView2 V1.0.781 is preventing other items on the form from refreshing, and showing parts of other tab which is behind in z order. Presume this is something to do with new DefaultBackgroundColor, which I haven't tried changing yet.

**Version**
SDK: 1.0.781-prerelease 
Runtime: 89.0.774.8 (Official build) dev (32-bit)
Framework: WinForms
OS: Win7 32

**Repro Steps**
I'm opening the same project as previous version. Other than Updating WebView2Loader.dll, Winforms.dll and Core.dll, I haven't made any code changes.

**Screenshots**
In this screenshot, the first tab (eBay) is at the front but the one behind seems to be showing through. This comes and goes when the content of the first tab changes. The colour of the 2nd tab hasn't changed to inactive.
![2021-01-30_113853](https://user-images.githubusercontent.com/30246320/106355668-6bca6a00-62f1-11eb-9a86-4b89b464021f.png)

**Additional context**

[AB#31562565](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/31562565)

**Comment by TF523** [comment]
@champnic
As an extension of this bug.  We installed this prerelease and the control caused a loss of functionality on our tab control page.  Tab required two clicks, otherwise controls were only half shown.  And all controls lost their onhover events...  We rolled back to previous pre-release...

Installed in windows VSTO addin taskpane tabbed page collection...

**Comment by champnic** [comment]
Thanks for the info. I've opened as a bug on our backlog and we'll take a look right away. @ukandrewc are you also using a specific tabbed control, or your own? If so, which control are you using?

**Comment by ukandrewc** [comment]
@champnic Not using a tabbed control, just adding WebView2 to form and bring to front when the Tab (a styled toolbar button) is clicked.

A Simplified example would be:

```
Dim Web1 As New WebView2
Dim Web2 As New WebView2

Sub Form_Load()
	'Bring first to front
	ToolstripBtn1.Click()
End Sub

ToolstripBtn1_Click()
	Web1.BringToFront()
End Sub

ToolstripBtn2_Click()
	Web2.BringToFront()
End Sub

```

Hope that gives you enough ;-)

**Comment by TF523** [comment]
> Thanks for the info. I've opened as a bug on our backlog and we'll take a look right away. @ukandrewc are you also using a specific tabbed control, or your own? If so, which control are you using?

@champnic 
We are using the standard tabbed page control from .net framework. This is the same vsto addin i have referenced previously...

When the tabbed page is single clicked the navigation controls are only partially visible. The tabbed page must be double clicked to show them properly. And the hover events for these controls no longer are active...

**Comment by AbeniMatteo** [comment]
@albahari found some [problem](https://github.com/MicrosoftEdge/WebView2Feedback/issues/878#issuecomment-770511476) with WM_PAINT, maybe it's related.

**Comment by champnic** [comment]
Thanks for finding this and linking the similar bug- I've opened a bug on our backlog and we're taking a look right away.

**Comment by ukandrewc** [comment]
Workaround in #878 fixes the controls on the form not refreshing, but not transparency issues with Z Order of WebView2 controls on the same form.

**Comment by champnic** [comment]
This should now be fixed in 1.0.790-prerelease SDK package. Thanks!
